client/src/components/App/Content/TabManagement/TabManagement.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Paper, Tabs, Tab, Card } from '@material-ui/core';
+
+import useStyles from './TabManagementStyles';
+
+const TabManagement: React.FC = (): JSX.Element => {
+    const classes = useStyles({});
+    const defaultTab = 0;
```

Comment:
Maybe it would be easier to save a tab as an object combined by title and id without keeping index and names separately.